projekt_do_pracy_na_labach/fiszki_backend/Dockerfile

```diff
@@ -1,2 +1,34 @@
 FROM python:3.10
 
+
+RUN mkdir /app/ 
+COPY . /app/
+
+WORKDIR /app 
+
+RUN pip install -r requirements.txt 
+
+CMD ["python", "main.py"]
+
+#Karol Jantas 
+#na 3.0
+#docker build --tag fiszki_frontend .
+#docker run fiszki_frontend 
+
+#na 3.5
+#docker run -p 8000:8000 fiszki_frontend 
+
+#na 4.0
+#docker network create siec1
+#docker run --network=siec1 fiszki_frontend
+#docker run --network=siec1 mongo
+#docker network inspect siec1
+
+#na 4.5 starsza wersja zadania 
+#docker volume create wolum1
+#docker run -v wolum1:/app fiszki_frontend
+#docker run -v wolum1:/app mongo 
```

Comment:
Mongo trzyma w innym miejscu dane. Miejsce gdzie znajdziesz informację jak poprawnie podmontować  znajdziesz na stronce https://hub.docker.com/_/mongo W sekcji "Where to store data"